Tangem/Modules/Send/Amount/Interactor/StakingSendAmountValidator.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  StakingSendAmountValidator.swift
+//  Tangem
+//
+//  Created by Dmitry Fedorov on 08.08.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import BlockchainSdk
+import Combine
+import TangemStaking
+
+class StakingSendAmountValidator {
+    private let tokenItem: TokenItem
+    private let validator: TransactionValidator
+    private let _minimumAmount = CurrentValueSubject<Decimal?, Never>(.none)
```

Comment:
хм, а на получается не нужен здесь `CurrentValueSubject`
может просто ?

```suggestion
    private var minimumAmount: Decimal?
```